docs/developer-guide/parallel-execution.md

```diff
@@ -0,0 +1,21 @@
+---
+description: StreamingFast Substreams parallel Execution
+---
+
+# Parallel Execution
+
+In simple terms, parallel execution is the ability to pre-compute the execution of Substreams modules. In more detail, parallel execution is the process of a Substreams module's code executing multiple segments of blockchain data simultaneously. Substreams modules are executed in parallel, rapidly producing data for consumption in end-user applications.
```

Comment:
`pre-compute` is not accurate enough because when doing backward parallel execution, we are not pre-computing.

Reply:
I've removed the initial sentence and left the rest of the content. The update will be in the next commit.